Document/0x06b-Basic-Security-Testing.md

```diff
@@ -15,7 +15,7 @@ The following is the most basic iOS app testing setup:
 - at least one jailbroken iOS device (of the desired iOS version)
 - Burp Suite or other interception proxy tool
 
-#### Setting up Xcode and Command Line Tools
+##### Setting up Xcode and Command Line Tools
```

Comment:
This can also be #### (4), right? Should be sub-point of iOS Testing Setup. It's also better style to not have just one title this deep (level 5)